### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by asifmehmood101**
in this PR i build last two section of my landing page Findus and footer sections

## here is figma link 
https://www.figma.com/file/yvClSI9AZBRX8UaaGEByF3/Modern-UI%2FUX%3A-Gericht?node-id=0%3A1

## Ticket link
https://app.clickup.com/t/21fhchk

### CODE REVIEW

File: src/container/Findus/FindUs.jsx [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+import { Container } from '../../components/Container';
+import { images } from '../../constants';
 
 const FindUs = () => (
-  <div>
-    FindUs
+  <div className="app__bg app__wrapper section__padding" id="contact">
+    <div className="app__wrapper_info">
+      <SubHeading>Contact</SubHeading>
+      <h1 className="headtext__cormorant" style={{ marginBottom: '3rem' }}>
```
Comment (Skrpk) [inline src/container/Findus/FindUs.jsx:11]: Why we can't set marginBottom property in .css file, but use inline style here?

File: src/container/Findus/FindUs.jsx [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+        Find Us
+      </h1>
+      <div className="app__wrapper-content">
+        <p className="p__opensans">
+          Lane Ends Bungalow, Whatcroft Hall Lane, Rudheath, CW9 75G
+        </p>
+        <p
+          className="p__cormorant"
+          style={{ color: '#DCCA87', margin: '2rem 0' }}
```
Comment (Skrpk) [inline src/container/Findus/FindUs.jsx:20]: Same here, we have class defined for this element.

File: src/App.css [hunk 3]
```diff
@@ -36,7 +36,6 @@
     flex: 1;
```
Comment (Skrpk) [inline src/App.css:36]: I tried to launch this app locally and got this error 

ERROR in ./src/components/SubHeading/SubHeading.jsx 5:0-26

"Module not found: Error: Cannot find file: 'Subheading.css' does not match the corresponding name on disk: './src/components/SubHeading/SubHeading.css'."

Are we aware of this?

File: src/container/Findus/FindUs.jsx (hunk 1 above)
Comment (asifmehmood101) [reply src/container/Findus/FindUs.jsx:11]: My bad , i was testing with inline styling

File: src/container/Findus/FindUs.jsx (hunk 2 above)
Comment (asifmehmood101) [reply src/container/Findus/FindUs.jsx:20]: same

**Pull request merged**